### PR TITLE
Add dydx page.

### PR DESCRIPTION
**Pull request by Eric-Warehime**
Adds a specific page for DYDX. 

Can separately add details for other chains afterwards.

### CODE REVIEW

File: docs/slinky/integrations/0-dydx.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+description: dYdX
+title: dYdX
+sidebar_position: 0
+---
+
+## FAQs
+
+### **Q: How do I get up and running?**
+
+**A:** Here is a quick (5-10 minutes) step by step breakdown. Please reach out to us at [skip.money/discord](http://skip.money/discord) if you have questions!
```
Comment (aljo242) [inline docs/slinky/integrations/0-dydx.md:11]:
```suggestion
**A:** Here is a quick (5-10 minutes) step-by-step breakdown. Please reach out to us at [skip.money/discord](http://skip.money/discord) if you have questions!
```

File: docs/slinky/integrations/0-dydx.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+   # PrometheusServerAddress is the address of the prometheus server that metrics will be
+   # exposed to.
+   prometheus_server_address = ""
+   ```
+
+### **Q: How do I know if my validator is properly fetching prices and posting them to the chain?**
+
+**A:** A full set of prometheus metrics are integrated into both the sidecar and the dYdX application binary.
+
+A comprehensive overview of the relevant metrics is hosted in the root [metrics.md](https://github.com/skip-mev/slinky/blob/main/metrics.md).
```
Comment (aljo242) [inline docs/slinky/integrations/0-dydx.md:95]: do we want this link from main or to be from a hash?

File: docs/slinky/integrations/0-dydx.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+`Failed to run fetch prices for slinky daemon`
+
+### **Q: How do I upgrade the sidecar binary?**
+
+**A:** Upgrading the sidecar can be done out of band of the chain’s binary. If you have a load balancer, CNAME, etc., in front of your sidecar you can simply start up the new version and switch out which version traffic is being directed to during live chain validation.
+
+If you are running the slinky sidecar in a container you can shut down the container, pull the updated container image and relaunch your container to update.
+
+If you are running the binary via systemd or other management tool you will need to stop the process and re-launch using the newly released binary.
```
Comment (aljo242) [inline docs/slinky/integrations/0-dydx.md:115]:
```suggestion
If you are running the binary via systemd or other management tool, you will need to stop the process and re-launch using the newly released binary.
```

File: docs/slinky/integrations/0-dydx.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+:::note
+We recommend you build some automation around config management either by pulling the latest `oracle.json` file directly from the release (if your dydx node is at localhost:1317) or via reconstructing the config using the `slinky-config` binary included in the release. The `oracle.json` file from previous releases will be compatible with future releases unless there is a major version bump, _however_, newly added price feeds may require updated information from your oracle config that was not present in the previous release which may cause breakage.
+:::
+
+The dydx node will still be able to participate in consensus without the sidecar, and will begin attaching prices to blocks once slinky is available. In the worst case, an upgrade in any of these manners will cause you to miss including vote extensions for a single block which should have no negative effects on you or the network.
+
+### **Q: Which version of the sidecar binary should I be running?**
+
+**A:** We are currently in the process of designing solutions around this problem in order to give validators reliable signal for exactly the minimum version of sidecar they need to be running.
```
Comment (aljo242) [inline docs/slinky/integrations/0-dydx.md:125]:
```suggestion
**A:** We are currently in the process of designing solutions around giving validators reliable signals for exactly the minimum version of sidecar they need to be running.
```

File: docs/slinky/integrations/0-dydx.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+In the short term, sidecar version bumps will be scheduled well in advance of any required upgrade and comms will be sent out in the relevant validator channels with the exact version required, and any changes which are relevant to the operator experience.
+
+In the future as we work with the chain and validators to build more automation around version bumps and reasoning about compatibility based on on-chain data, we plan to increase the cadence of updates in line with feature additions, as opposed to batching for larger releases on a longer timeline.
```
Comment (aljo242) [inline docs/slinky/integrations/0-dydx.md:129]: this sentence is too long

**Review by aljo242 — COMMENTED** [review]
Sometimes we use "slinky" and sometimes we use "Slinky" 

Let's use "Slinky" consistently

File: docs/slinky/integrations/0-dydx.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+**A:** Here is a quick (5-10 minutes) step by step breakdown. Please reach out to us at [skip.money/discord](http://skip.money/discord) if you have questions!
+
+1. **Download the slinky binary.**
+
+   The best way to get the slinky binary is in the GitHub releases page for Slinky.
+
+   https://github.com/skip-mev/slinky/releases
+
+   The initial version required for dYdX is `0.3.1`
```
Comment (nivasan1) [inline docs/slinky/integrations/0-dydx.md:19]: Is the plan for us to update this everytime a new slinky version is pushed to dydx (i.e 0.4.0)?

File: docs/slinky/integrations/0-dydx.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+
+   This will include the Slinky binary, `slinky` , and a utility binary `slinky-config` as well as some pre-generated configuration files.
+
+2. **Integrate the Slinky sidecar into your setup.**
+
+   The configuration of your validator setup requires you to tackle a few problems which we’ll mention here.
+
+   **_Configure the Slinky process_**
+
+   Slinky has 1 important config file for running with dYdX.
```
Comment (aljo242) [inline docs/slinky/integrations/0-dydx.md:33]:
```suggestion
   Slinky has 1 important config file for running with dYdX:
```

File: docs/slinky/integrations/0-dydx.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+   Because the set of prices the chain wants to fetch change frequently, you must run Slinky with dynamic config updating. Using the `slinky-config` binary you can generate an `oracle.json` file which watches the chain and updates Slinky when on-chain data is changed.
+
+   As an example, running the following would point your sidecar binary at the `dydxprotocold` node binary running on the same host at the default port of `1317`:
+
+   ```bash
+   slinky-config --chain dydx --node-http-url "localhost:1317"
+   ```
+
+   This command (with default localhost:1317) should produce the equivalent of the `oracle.json` file bundled in the `config/dydx` directory in the release. After running this command you should have produced valid `oracle.json` file and you can start your sidecar process.
+
```
Comment (nivasan1) [inline docs/slinky/integrations/0-dydx.md:50]: Maybe useful to include the relative path of the output file

**Review by aljo242 — APPROVED** [review]
lgtm with mini nit

File: docs/slinky/integrations/0-dydx.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+**A:** Upgrading the sidecar can be done out of band of the chain’s binary. If you have a load balancer, CNAME, etc., in front of your sidecar you can simply start up the new version and switch out which version traffic is being directed to during live chain validation.
+
+If you are running the Slinky sidecar in a container you can shut down the container, pull the updated container image and relaunch your container to update.
+
+If you are running the binary via systemd or other management tool, you will need to stop the process and re-launch using the newly released binary.
+
+:::note
+We recommend you build some automation around config management either by pulling the latest `oracle.json` file directly from the release (if your dydx node is at localhost:1317) or via reconstructing the config using the `slinky-config` binary included in the release. The `oracle.json` file from previous releases will be compatible with future releases unless there is a major version bump, _however_, newly added price feeds may require updated information from your oracle config that was not present in the previous release which may cause breakage.
+:::
```
Comment (nivasan1) [inline docs/slinky/integrations/0-dydx.md:119]: nit would explicitly say which oracle.json to use, i.e in `config/dydx`

**Review by nivasan1 — APPROVED** [review]
lgtm w/ nits

File: docs/slinky/integrations/0-dydx.md (hunk 6 above)
Comment (Eric-Warehime) [reply docs/slinky/integrations/0-dydx.md:19]: For now I think it's fine--can add to release documentation.

**Pull request merged**